### PR TITLE
Tooltip XSS for LF8

### PR DESCRIPTION
**Pull request by GuillaumeNury**
## Fix / Breaking Changes

* `luTooltip` no longer bypass Angular DOM sanitizer. Some tooltip may loose some elements.

## Features

* `luTooltip` accepts `SafeHtml | string` so you can call `bypassSecurityTrustHtml` on your own
* `luSafeHtml` offers a safer way of using `bypassSecurityTrustHtml`. It uses [dompurify](https://github.com/cure53/DOMPurify) under the hood in order to have a more flexible dom sanitizer.

  ```html
  <div [luTooltip]="someHtml | luSafeHtml:{ ALLOWED_TAGS: ['br', 'strong', 'em', 'ul', 'li'] }"></div>
  ```

### CODE REVIEW

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-185-f706469a-7731-4008-9ce5-22b612578ef2/ui-diff.html)

File: packages/ng/tooltip/src/lib/trigger/tooltip-trigger.directive.spec.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { LuTooltipModule } from '../tooltip.module';
+import { LuTooltipTriggerDirective } from './tooltip-trigger.directive';
+
+@Component({
+	standalone: true,
+	imports: [LuTooltipModule],
+	selector: 'lu-test',
+	template: `<div id="with-tooltip" [luTooltip]="content"></div>`,
+})
```
Comment (shajz) [inline packages/ng/tooltip/src/lib/trigger/tooltip-trigger.directive.spec.ts:13]:
```suggestion
@Component({
	imports: [LuTooltipModule],
	selector: 'lu-test',
	template: `<div id="with-tooltip" [luTooltip]="content"></div>`,
})
```
ng 14 only 😁

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1763/storybook

**Review by shajz — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-188-c88d8e43-dc97-4c27-9175-6f207efb66f7/ui-diff.html)

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-189-0f6005f0-16d0-4237-91a2-e9fda8963525/ui-diff.html)

**Pull request merged**